core/src/apps/cardano/helpers/credential.py

```diff
@@ -221,8 +228,9 @@ def _do_base_address_credentials_match(
     address_n: list[int],
     address_n_staking: list[int],
 ) -> bool:
-    return address_n_staking == _path_to_staking_path(address_n)
-
+    from .paths import CHAIN_STAKING_KEY
+    from .utils import to_account_path
 
-def _path_to_staking_path(path: list[int]) -> list[int]:
-    return to_account_path(path) + [CHAIN_STAKING_KEY, 0]
+    # _path_to_staking_path
```

Comment:
please remove this comment

Reply:
Deleted in [f38408b](https://github.com/trezor/trezor-firmware/pull/2525/commits/f38408bc5e551a04efb3b56a35ec4c2918b53137)